rubygem-openscap/rubygem-openscap.spec

```diff
@@ -66,32 +60,25 @@ BuildArch: noarch
 Development files for %{name}
```

Comment:
I can't comment on the specific line, but there's an Obsoletes in here that needs the %{?scl:} conditional removing, otherwise it's not going to evaluate in a nonscl buildroot.

Like above, it needs to obsolete both the ruby193- and tfm- variants.